tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "allowJs": true,
+        "baseUrl": ".",
+        "experimentalDecorators": true,
```

Comment:
Does our code actually use decorators? I didn't see any. Or is there a technical reason why this has to be set?

Reply:
Honestly, I just copied the tsconfig,json we use for all our project templates. Let's keep it the same, as we utlimately will merge from the Office-Addin-Taskpane project template and keeping it the same across the board obviates any merge conflicts that might occur

Reply:
OK. But we should revisit this someday. In principle, I don't really like the idea of doing things that are convenient to us, but potentially confusing to our customers. But I understand that sometimes its the lesser of evils.